docs/en/testnet.md

```diff
@@ -68,51 +52,51 @@ DEV.preDymaxion.startBlock = 0
 DEV.poc2.startBlock = 0
 DEV.rewardRecipient.startBlock = 6500
 ```
+Keep in mind there may be scenarios where you would like to have your own private test net, say in case of some future revisions that might require a fork for adding to the current chain. In this case, you could simply change the value of DEV.P2P.BootstrapPeers to the value of your localhost.
 
-If you just need testnet wallet for testing please use this [link](http://3.16.150.48:6876/index.html#). If you want to use your own desired peer, you can just change ip addresses.
+If you just need to communicate with another TestNet peer, you can use this [link](http://3.16.150.48:6876/index.html#). If you want to use your own desired peer, you can just change the ip address accordingly.
 
-You can get the testnet coins at this [faucet](http://burstcoin.cc:7777/)
+You can get TestNet Burst coins at this [faucet](http://burstcoin.cc:7777/).
 
-If for any reason you need to reset your peer, a fast easy way to do it is as follows:
+If for any reason you need to reset your peer, here is a fast and easy way to do it:
```

Comment:
If using the default H2 database backend (which is the default), just remove all files under the `burst_db` folder. If using mariaDB, here is a fast and easy way to do it: